### DISCUSSION TITLE
How to get the current cursor position ?

### DISCUSSION THREAD

**Discussion by usman0011** [xdan/jodit#826] [Q&A] [answered]
I've looked around in the docs a lot but can't seem to find any method or class that gives this info. Any help will be greatly appreciated. Thank you

**Comment by xdan** [comment, marked as answer]
Use the selection API on `editor.s` (alias of `editor.selection`). For the raw caret, `editor.s.range` gives you the current `Range`, so `range.startContainer` / `range.startOffset` is the cursor position; `editor.s.current()` returns the node at the cursor. If you need to remember the spot and come back to it, `editor.s.save()` / `editor.s.restore()` drop and reuse markers.
